resources/views/entities/pages/entity-notes/edit.blade.php

```diff
@@ -40,4 +44,39 @@
     @parent
     {!! Form::open(['method' => 'DELETE', 'route' => ['entities.entity_notes.destroy', 'entity' => $entity, 'entity_note' => $model], 'style' => 'display:inline', 'id' => 'delete-form-note-' . $model->id]) !!}
     {!! Form::close() !!}
+
+    @if(!empty($editingUsers) && !empty($model))
```

Comment:
We can just define all this HTML only once and use an include to avoid copy-pasting :)